packages/components/Tabs/src/Tabs.tsx

```diff
@@ -95,7 +96,7 @@ export const Tabs = compose<TabsType>({
 
   settings,
   slots: {
-    root: View,
+    root: Stack,
```

Comment:
What does the change from View to Stack do?

Reply:
I don't remember why I changed that lol. To be honest I'm not even sure what Stack does in general.

Reply:
We should probably check the difference before changing it.